### PR TITLE
[Indexer fix] exitorrent.org dynamic javacript download urls

### PR DESCRIPTION
**Pull request by relains**
Few days ago indexer changed all urls to dynamically generated with javascript.

### CODE REVIEW

**Comment by garfield69** [comment]
None of the team have an account on exitorrent so we cannot test independently.
Have you tested the indexer you've updated by doing searches, downloads, and checking all the fields are populated correctly?

File: src/Jackett.Common/Definitions/exitorrent-org.yml [hunk 1]
```diff
@@ -104,7 +104,7 @@ search:
       args: [" +(?:19|20)\\d{2} *$", ""]
 
   rows:
-    selector: table[class="text"][border="0"][cellspacing="0"][cellpadding="4"] > tbody > tr:has(a[href^="details.php?id="]):has(td[class*="tableBody1"])
+    selector: table[class="text"][border="0"][cellspacing="0"][cellpadding="4"] > tbody > tr:has(a[href^="details.php?id="]):has(td[class*="tableBody1"], td[class*="tableBody1B"])
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/exitorrent-org.yml:107]: if the last selector does the job, do we need the first selector anymore?

Comment (relains) [reply src/Jackett.Common/Definitions/exitorrent-org.yml:107]: On the site there are multiple different td-s that has class tableBody1 or tableBody1B and this is the only differentiator from others. They are used interchangeably so when I get rid of one there may be some results missing. Maybe You could do it better but I found it work OK.

**Comment by relains** [comment]
> None of the team have an account on exitorrent so we cannot test independently. Have you tested the indexer you've updated by doing searches, downloads, and checking all the fields are populated correctly?

Of courde I've tested it :) I'm using this tracker on daily basis so i really want it to work :)

**Comment by garfield69** [comment]
Thank you.

**Comment by garfield69** [comment]
v0.22.56

**Pull request merged**